utils/helpers.ts

```diff
@@ -38,9 +38,22 @@ export function getTargetNumber(target:String){
 }
 
 export function isLString(element){
-    return  element.hasOwnProperty('ar') || element.hasOwnProperty('en') ||
-            element.hasOwnProperty('fr') || element.hasOwnProperty('es') || 
-            element.hasOwnProperty('ru') || element.hasOwnProperty('zh');
+    let hasLocale =   element.hasOwnProperty('ar') || element.hasOwnProperty('en') ||
+                      element.hasOwnProperty('fr') || element.hasOwnProperty('es') || 
+                      element.hasOwnProperty('ru') || element.hasOwnProperty('zh');
+                      
+    if(!hasLocale){
+        for (const key in element) {
+            if (element.hasOwnProperty(key)) {
+                if (key.length == 2) {
+                    hasLocale = true;
+                    break;
+                }
+            }
+        }
+    }
+    
+    return hasLocale;
```

Comment:
what about using object.keys() with some or every?
```suggestion
    return Object.keys(element).some(l=>/^[a-z]{2,3}$/.test(l)); 
```